addons/account/models/account_journal.py

```diff
@@ -571,13 +571,13 @@ def write(self, vals):
         return result
 
     @api.model
-    def get_next_bank_cash_default_code(self, journal_type, company):
-        journal_code_base = (journal_type == 'cash' and 'CSH' or 'BNK')
+    def get_next_bank_cash_default_code(self, journal_type, company, codes=None):
```

Comment:
I couldn't find a way around modifying the function signature in stable without adding a lot of duplicate code